hedera-node/hedera-app/src/main/java/com/hedera/node/app/Hedera.java

```diff
@@ -804,9 +805,10 @@ private void initializeDagger(@NonNull final MerkleHederaState state, @NonNull f
                             throttleManager,
                             exchangeRateManager,
                             monoMultiplierSources,
-                            handleThrottling))
+                            throttleAccumulator))
                     .networkUtilizationManager(
-                            new NetworkUtilizationManagerImpl(handleThrottling, monoMultiplierSources))
+                            new NetworkUtilizationManagerImpl(throttleAccumulator, monoMultiplierSources))
+                    .synchronizedThrottleAccumulator(new SynchronizedThrottleAccumulator(throttleAccumulator))
```

Comment:
I think we need a separate instance of `ThrottleAccumulator` for the synchronized one with capacity split equal to the number of nodes (i.e. the FE throttling), leaving the current `throttleAccumulator` with capacity split of 1 to be the one used in the handle workflow and congestion (i.e. the BE throttling).
You can check [the mono setup](https://github.com/hashgraph/hedera-services/blob/25bf841331b46c748a081b0f01a1ad1e86cb21e4/hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/throttling/ThrottlingModule.java#L33) for reference.

P.S. Having separate instances for FE and BE throttle would mean we need to call `rebuildFor` and `applyGasConfig` methods for both.